app/scripts/modules/appengine/serverGroup/configure/wizard/cloneServerGroup.controller.ts

```diff
@@ -33,34 +33,21 @@ class AppengineCloneServerGroupCtrl {
               private $uibModalInstance: any,
               public serverGroupCommand: IAppengineServerGroupCommand,
               private application: Application,
-              taskMonitorBuilder: TaskMonitorBuilder,
+              private taskMonitorBuilder: TaskMonitorBuilder,
               private serverGroupWriter: ServerGroupWriter,
               commandBuilder: AppengineServerGroupCommandBuilder) {
-
     if (['create', 'clone', 'editPipeline'].includes(get<string>(serverGroupCommand, 'viewState.mode'))) {
       $scope.command = serverGroupCommand;
       this.state.loading = false;
+      this.initialize();
```

Comment:
`this.initialize()` happens in both branches of this `if` I believe, it could go after.